libraries/shared/src/WarningsSuppression.h

```diff
@@ -0,0 +1,53 @@
+//
+//  WarningsSuppression.h
```

Comment:
Filename may make more sense as "WarningSuppression".

Reply:
There's potentially more than one warning that would be suppressed. Just so far it's enough to have one.